<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>38</y>
      <width>831</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Processos</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>821</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>PID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditPID_4"/>
       </item>
       <item>
        <widget class="QPushButton" name="botaoSuspender_4">
         <property name="text">
          <string>Suspender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoContinuar_4">
         <property name="text">
          <string>Continuar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoMatar_4">
         <property name="text">
          <string>Matar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoAlterarCPU_4">
         <property name="text">
          <string>Alterar CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="botaoMudarPrioridade_4">
         <property name="text">
          <string>Mudar Prioridade</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTableWidget" name="tabela_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>741</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <column>
       <property name="text">
        <string>%CPU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PRIORIDADE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>%MEM</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>COMANDO</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CPU</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Filtro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditFiltro_4"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_15">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Prioridade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxPrioridade_4">
         <item>
          <property name="text">
           <string>-20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_16">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>50</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>CPU</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxCPU_4">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QProgressBar" name="CPU0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>118</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GERENCIADOR DE PROCESSOS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>botaoSuspender_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>suspender()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoContinuar_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>continuar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoMatar_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>matar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoAlterarCPU_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>alterarCPU()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botaoMudarPrioridade_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mudarPrioridade()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditFiltro_4</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>filtro()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabela_4</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>pegarPIDTabela(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>812</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>suspender()</slot>
  <slot>continuar()</slot>
  <slot>matar()</slot>
  <slot>alterarCPU()</slot>
  <slot>mudarPrioridade()</slot>
  <slot>filtro()</slot>
  <slot>pegarPIDTabela(int,int)</slot>
 </slots>
</ui>
